### PR TITLE
Closes #1173 close again

### LINKED ISSUE #1173: Update the PR guidance to include closes keyword to auto-close issues
### Feature Idea

In https://pharmaverse.github.io/admiral/articles/git_usage.html#pull-request add guidance that the PR title must include keyword "closes #_issuenumber_", e.g. see example below:
image

so that the issue will auto-close when the PR updates get merged with main later.

A reminder on this can also be added to the PR checklist template at : https://github.com/pharmaverse/admiral/blob/main/.github/pull_request_template.md, and the `Close #<insert_issue_number>` can be removed from this.

### Relevant Input

N/A

### Relevant Output

N/A

### Reproducible Example/Pseudo Code

N/A

### PR DESCRIPTION
**Pull request by bms63**
Closes #1173 

Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/) 
- [x] Updated relevant unit tests or have written new unit tests - See [Unit Test Guide](https://pharmaverse.github.io/admiral/articles/unit_test_guidance.html#writing-unit-tests-in-admiral-)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiral/articles/programming_strategy.html#deprecation-1)?
- [x] Update to all relevant roxygen headers and examples 
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue so that it closes after successful merging. 
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Review by dinakar29 — APPROVED** [review]
This is great. Thanks, @bms63!

**Comment by bms63** [comment]
Ummm...maybe we need to change this line 

- [ ]  Link the issue so that it closes after successful merging.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
-Close #<insert_issue_number>
+Closes #<insert_issue_number>
```
Comment (rossfarrugia) [inline .github/pull_request_template.md:1]: i would delete this line altogether. fine to only include in the PR title now and not repeat it twice

**Comment by rossfarrugia** [comment]
> Ummm...maybe we need to change this line
> 
> * [ ]   Link the issue so that it closes after successful merging.

@bms63 agree - i'd just delete "so that it closes after successful merging" - i still like the convention to link issues and PRs as it helps when looking back at older issues

**Comment by rossfarrugia** [comment]
@bms63 do we have anywhere in our PR guidance that states what we're trying to achieve here... i.e. a better worded version of what i put in the issue... 

_we are trying to achieve that the issue stays open for transparency even if a PR has already fixed it in devel until the point we make a release and merge it to main and then the issue should auto-close. otherwise we often get repeat issues raised for things already covered by another issue, just not yet released._

**Review by dinakar29 — APPROVED** [review]

**Review by rossfarrugia — APPROVED** [review]
nice job @bms63 - thanks!

**Pull request merged**